os_dep/linux/ioctl_cfg80211.c

```diff
@@ -3651,7 +3669,13 @@ static int rtw_cfg80211_add_monitor_if(_adapter *padapter, char *name,
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(4,1,0))
 	mon_ndev->name_assign_type = name_assign_type;
 #endif
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(4,11,9))
+	mon_ndev->needs_free_netdev = true;
```

Comment:
Should be `false` as `rtw_ndev_destructor()` will call `free_netdev()`: https://github.com/abperiasamy/rtl8812AU_8821AU_linux/blob/a12fc996d3d153b566788d7f858f77dbdd07a4f6/os_dep/linux/os_intfs.c#L2691